routes/notif_nodemailer.js

```diff
@@ -23,9 +23,9 @@ if(CONFIG.nodemailer){
 }
 
 
-function verify_transport () {
+async function verify_transport () {
     // eslint-disable-next-line no-unused-vars
-    transporter.verify(function(error, success) {
+    await transporter.verify(function(error, success) {
```

Comment:
as there is an await here, there should be no callback function here, no? something like

    try {
        await transporter.verify()
        mail_set = true
        ....
    } catch() {

   }

and catch possible error , set mail_set etc... if success

Reply:
https://github.com/genouest/genouestaccountmanager/pull/166/commits/2dcd5caf9dac371d019970ebc5d1c7dc20e3c4cf